src/chrome/content/rules/OPM.gov.xml

```diff
@@ -116,15 +115,18 @@
 	<!--	Complications:
 				-->
 	<target host=    "www.leadership.opm.gov" />
+	
+	<!-- Unable to connect via both HTTP and HTTPS
 	<target host=    "servicesonline.opm.gov" />
+	-->
 
 
 	<securecookie host="." name="." />
 
 
-	<!-- could not resolve host -->
+	<!-- could not resolve host 
 	<rule from="^http://servicesonline\.opm\.gov/"
-		to="https://www.servicesonline.opm.gov/" />
+		to="https://www.servicesonline.opm.gov/" />-->
```

Comment:
This seems no longer to be the case.  I'm able to access https://www.servicesonline.opm.gov/ fine.